packages/templates/wasm/assemblyscript/.gitignore

```diff
@@ -2,3 +2,4 @@ build
 node_modules
 wrap
 .polywrap
+./workflows/output.json
```

Comment:
Why are we adding this to gitignore? I think it should be up to dev, whether they want to ignore it or not. This may not even be output.json and could be named something else so I don't see any point in ignoring it.

Reply:
i removed it